patches/ui-views-controls-button-md_text_button.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ui/views/controls/button/md_text_button.h b/ui/views/controls/button/md_text_button.h
+index 5fd2ef10bc43f350c7424daad5aafbac0ca96720..f68362f1558667f2268494131157abff4f0608ce 100644
+--- a/ui/views/controls/button/md_text_button.h
++++ b/ui/views/controls/button/md_text_button.h
+@@ -41,7 +41,7 @@ class VIEWS_EXPORT MdTextButton : public LabelButton {
+ 
+   ~MdTextButton() override;
+ 
+-  void SetStyle(ui::ButtonStyle button_style);
++  virtual void SetStyle(ui::ButtonStyle button_style);
```

Comment:
MdTextButton has its Builder `BEGIN_VIEW_BUILDER(VIEWS_EXPORT, MdTextButton, LabelButton)`.
and that builder has `SetStyle` method `VIEW_BUILDER_PROPERTY(ui::ButtonStyle, Style)`
`#define SetStyle virtual SetStyle` affects this builder's interface also. so directly patched.

Reply:
I don't understand why we need to override `SetStyle` at all just to set some `Kind` thing that maps directly to the style. Why aren't we just using the style directly?